handlers/readme.md

```diff
@@ -0,0 +1,52 @@
+# Open Growth Signals and Delights
+
+**Signal** - An indication of a task that needs to be done automatically using Open Growth
+
+**Delight** - A method of outreach to your customer i.e. email, text, tweet, special delivery of cake :cake:
+
+The core vision of Open Growth is to automate tedious workforce tasks and give humans more time to work on the big picture. Open Growth’s **Signals** and **Delights** are an excellent resource to automate email correspondence with your vast customer base. A lone developer can get an automatic email system up and running in a few minutes using Open Growth. We will explain how to set up an automatic email to your newest customers when they sign up for your service.
+
+First, there needs to be a source of the emails you send. We will call this a **Signal Generator**. The Signal Generator needs to collect customer information, like email address and name (if available), and publish the Signal to Open Growth. This can be accomplished by a program that automatically runs every so often using a service like [Cron](https://en.wikipedia.org/wiki/Cron). The program can query your customer database for any new sign ups, and publish a message to your PubNub Open Growth instance immediately. 
```

Comment:
May want to mention along side cron that it can be done using PubNub directly in their app/site/whatever